recipes/peco/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set org = "peco" %}
+{% set name = "peco" %}
+{% set version = "0.5.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8e32c8af533e03795f27feb4ee134960611d2fc0266528b1c512a6f1f065b164
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('go') }}
```

`compiler('go')` is outdated and should no longer be used.

```suggestion
    - {{ compiler('go-nocgo') }}
```

---

Thanks, I committed it like you revised.